cves/kernel/CVE-2020-25284.yml

```diff
@@ -325,9 +317,9 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: I could not find any discussion on commits, mailing lists, or reports.
```

Comment:
There looks to be a few mailing lists on this CVE. Unsure if you wish to include them.

1. https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/message/TH734NYI5EZD4XQ2VE5Y7HNPWOK3EVXA/

2. https://lists.debian.org/debian-lts-announce/2020/10/msg00032.html